src/main/java/com/puppycrawl/tools/checkstyle/GlobalStatefulCheck.java

```diff
@@ -37,7 +37,7 @@
  * Checker guarantees that there will be exactly one check instance
  * This is similar to multi-file validation, which checkstyle does not support fully yet.
  * Please refer to https://github.com/checkstyle/checkstyle/issues/3540 for details.
- * @noinspection AnnotationClass, ClassIndependentOfModule, unused
+ * @noinspection AnnotationClass, ClassIndependentOfModule, unused, ClassOnlyUsedInOnePackage
```

Comment:
unused and class only in one package should go away, right?

Reply:
eventually yes, IDEA keep complaining on them so for now they should stay.